docs/governance/politeia/example-proposals.md

```diff
@@ -0,0 +1,156 @@
+# <img class="dcr-icon" src="/img/dcr-icons/Politeia.svg" /> Example Politeia Proposals
+
+---
+
+Below are two example Politeia proposals that utilize the template provided in the
+Politeia [proposal guidelines](/proposal-guidelines.md). The first example proposes the 
```

Comment:
Remove the slash from the link path - `(proposal-guidelines.md)`

Reply:
Done